images/s6_assets/pulp_tests.sh

```diff
@@ -12,16 +12,25 @@ echo "Base Address: $BASE_ADDR"
 grep "127.0.0.1   pulp" /etc/hosts || echo "127.0.0.1   pulp" | sudo tee -a /etc/hosts
 
 echo "Installing Pulp-CLI"
-if [ -d pulp-cli ]; then
-  cd pulp-cli
-  git fetch origin
-  git reset --hard origin/main
-else
+pip install pulp-cli
+
+# Retreive installed pulp-cli version
+PULP_CLI_VERSION=$(python3 -c \
+  'import importlib.metadata; \
+   from packaging.version import Version; \
+   print(Version(importlib.metadata.version("pulp-cli")))')
+
+# Checkout git repo for pulp-cli at correct version to fetch tests
+if [ ! -d pulp-cli ]; then
   git clone --depth=1 https://github.com/pulp/pulp-cli.git
```

Comment:
```suggestion
git clone --depth=1 https://github.com/pulp/pulp-cli.git -b "${PULP_CLI_VERSION}"
```
You should be able to clone that tag directly.

Reply:
Thanks. I have made the change